cd/mxnet_lib/static/Jenkins_pipeline.groovy

```diff
@@ -26,7 +26,7 @@
 // NOTE: the following variables are referenced in the mxnet_lib_pipeline jenkins file imported bellow
 
 // libmxnet location
-libmxnet = 'lib/libmxnet.so'
+libmxnet = 'lib/libmxnet.so*'
```

Comment:
With this change, are we planning to package all three binaries (viz. libmxnet.so, libmxnet.so.2, libmxnet.so.2.0.0) into the nightly wheel? @sl1pkn07 @leezu